server/mock_helpers_test.go

```diff
@@ -0,0 +1,40 @@
+package server_test
+
+import (
+	"fmt"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+)
+
+type mockSequence struct {
+	first, last *gomock.Call // may be both nil (= the default value of mockSequence) to mean empty sequence
+}
+
+// like gomock.inOrder, but can be nested
+func inOrder(calls ...interface{}) mockSequence {
```

Comment:
This is a copy of code from `internal/storage`; is there a good place to share it?

Reply:
How about creating a new package in `internal` for this?

Reply:
Moved to `internal/mockutils`.